### PR TITLE
Add qsum recipe

### PR DESCRIPTION
**Pull request by QCoding**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/qsum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+source:
+  url: https://github.com/QCoding/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 'e5c01285550d9df96568b12e711d52074263d42f0ba4078dd3fd4aff69c201f6'
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/qsum/meta.yaml]: For packages that don't use compilers, you don't need the `build` section, just `host` and `run`, so please remove `build` from here.

File: recipes/qsum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - python >=3.5.0
+    - setuptools
+  host:
+    - python >=3.5
+    - setuptools
```
Comment (npavlovikj) [inline recipes/qsum/meta.yaml]: I don't think you need setuptools to build the packages, so please remove it.

File: recipes/qsum/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% set name = "qsum" %}
+  {% set version = "0.2.0" %}
```
Comment (npavlovikj) [inline recipes/qsum/meta.yaml]: Remove spaces.

File: recipes/qsum/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/QCoding/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 'e5c01285550d9df96568b12e711d52074263d42f0ba4078dd3fd4aff69c201f6'
+
+build:
+  noarch: python
+  number: 0
+
```
Comment (npavlovikj) [inline recipes/qsum/meta.yaml]: For `noarch` packages, you can remove the `bld.bat` and `build.sh` files, and add `script: "{{ PYTHON }} -m pip install . -vv"` here in the `script` section, so please do that.

File: recipes/qsum/bld.bat [hunk 5]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (npavlovikj) [inline recipes/qsum/bld.bat]: Remove this file.

File: recipes/qsum/build.sh [hunk 6]
```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install     # Python command to install the script.
```
Comment (npavlovikj) [inline recipes/qsum/build.sh]: Remove this file.

**Comment by npavlovikj** [comment]
I added few comments. Additionally, looks like you have deleted `recipes/example/meta.yaml`, so please don't remove the example files and reverse that commit.

**Comment by npavlovikj** [comment]
The recipe looks great @QCoding  - thank you for the prompt fixes and contributing 🎉

**Comment by QCoding** [comment]
@npavlovikj thanks for the review!

**Pull request merged**